types/react-sortable-tree/index.d.ts

```diff
@@ -184,7 +183,7 @@ interface ThemeTreeProps {
     reactVirtualizedListProps?: Partial<ListProps>;
     scaffoldBlockPxWidth?: number;
     slideRegionSize?: number;
-    rowHeight?: ((info: Index) => number) | number;
+    rowHeight?: ((info: TreeIndex & TreeNode & TreePath) => number) | number;
```

Comment:
`TreeIndex & TreeNode & TreePath` can be shortened as `NodeData`
And according to https://github.com/frontend-collective/react-sortable-tree/blob/master/src/react-sortable-tree.js#L714 + https://github.com/frontend-collective/react-sortable-tree/blob/master/src/react-sortable-tree.js#L746 it should be `NodeData & Index`

Reply:
That does look cleaner! I've updated the typing to use `NodeData & Index`